_posts/2017-08-08-euroclojure-2017.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: post
+title: EuroClojure 2017
+author: mkainula, tatut, pkalliok, viesti
+excerpt: A brief description of the presentations held in EuroClojure 2017 in Berlin.
+tags:
+- Clojure
+---
+
+EuroClojure 2017 was held in Berlin, Germany on July 20th and 21st. The venue was Kulturbrauerei, an old brewery turned into a cultural center, in the lively Prenzlauer Berg neighborhood. This year, there were seven Clojure enthusiasts from Solita attending the conference, and a couple more from our subsidiary Evolta.
+
+This post contains a quick recap of the talks and tips on which talks you should watch from Youtube (once they are posted there)
+
+![bunch of Solita people](/img/euroclojure2017/solita.jpg)
+
+
+## Day 1
+
+### otplike – Erlang/OTP processes and behaviours for Clojure, Alexey Aristov
+
+The first talk of the conference was about bringing some of the concepts and proven design patterns for building large scale systems from Erlang/OTP to Clojure with otplike library. It uses core.async and core.match under the hood, and if you are already familiar with Erlang/OTP patterns, you can use them in similar way. It’s easy to run into pitfalls using the low level core.async itself, so the ideas of processes, process linking and supervision are helpful building blocks and help in avoiding those pitfalls.
+
+However, not all of the features from Erlang/OTP can be implemented, due to the limitations of JVM. There is also work in progress for a ClojureScript version of the library.
```

Comment:
joko "due to JVM limitations" tai "due to the limitations in JVMs"

Sanajärjestys:  There _also is_  https://english.stackexchange.com/questions/28950/i-ve-been-also-or-ive-also-been